consent/migrations/sql/shared/14.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+ALTER TABLE hydra_oauth2_consent_request ADD handled_at timestamp NULL;
```

Comment:
What do you think about `created_at` and `updated_at`?

Reply:
We decided to name this field `handled_at` because:
- `created_at` could also be the date when the database entry was created
- `updated_at` could also be a change of any other parameter in the request

`handled_at` is more explicit, therefore we named the new field `handled_at`

Reply:
That makes sense!